index.html

```diff
@@ -123,18 +123,26 @@ <h4 class="panel-title">
                         Save password in your browser
                     </label>
                   </div>
+                  <div class="checkbox">
+                    <label class="control-label" for="ssl">
+                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
+                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    </label>
+                  </div>
                   <div class="checkbox" ng-show="settings.savepassword">
                     <label class="control-label" for="autoconnect">
-                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect">
+                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect"  ng-disabled="settings.useTotp">
                         Automatically connect
                     </label>
                   </div>
                   <div class="checkbox">
-                    <label class="control-label" for="ssl">
-                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
-                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    <label class="control-label" for="useTotp">
+                      <input type="checkbox" id="useTotp" ng-model="settings.useTotp" ng-change="changeUseTOTP()">
```

Comment:
you know you can add a callback for when a setting changes, too? https://github.com/glowing-bear/glowing-bear/blob/3d042b125bd40f51f4d9cc53fa9abb801f11cd71/js/settings.js#L52-L65

Reply:
Using that is a bit cheaper on the CPU and more robust because it also triggers if any other code changes the setting, not just when the checkbox is changed

Reply:
I did not know, thanks. Changed.